src/fingerprint.rs

```diff
@@ -94,7 +95,7 @@ fn load_all_fingerprints_built_with(
             }
         }
     }
-    debug!("Hashs to keep: {:#?}", keep);
+    trace!("Hashs to keep: {:#?}", keep);
```

Comment:
The `--verbose` output used to be unreadable due to the pollution of the hashes listing.

Now these hashes are behind the TRACE level, and it can be enabled with `-v -v` or `-vv` (`--verbose` twice).